app/helpers/reviews_helper.rb

```diff
@@ -0,0 +1,29 @@
+module ReviewsHelper
+
+  def reviews_find_movie
+
+  end
+
+  def find_reviews_by_user
+    if user_signed_in?
+      current_user.reviews
+    end
+  end
+
+  def has_already_reviewed?
+    if !user_signed_in?
+      return true
+    else
+      find_reviews_by_user.any?{|r| r.movie_id == params[:movie_id]}
```

Comment:
Let's do this with SQL via Active Record. Docs [here](http://apidock.com/rails/ActiveRecord/FinderMethods/exists%3F).